testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -890,13 +905,17 @@ fn forked_tenure_testing(
         |config| {
             // make the duration long enough that the reorg attempt will definitely be accepted
             config.first_proposal_burn_block_timing = proposal_limit;
+            // don't allow signers to post signed blocks (limits the amount of fault injection we
+            // need)
+            config.broadcast_signed_blocks = false;
         },
         |_| {},
         &[],
     );
     let http_origin = format!("http://{}", &signer_test.running_nodes.conf.node.rpc_bind);
 
     signer_test.boot_to_epoch_3();
+    sleep_ms(1000);
```

Comment:
Same comment as elsewhere, if a sleep is necessary, it should be replaced with `wait_for()` and the condition.